### PR TITLE
Add issue templates, footer and contact us page

### PR DESCRIPTION
**Pull request by davidhunter08**
- add GitHub issue templates (.yaml files)
- add global footer
- add service name to header
- add 'Help and feedback' page
- adjust home page hero spacing

### CODE REVIEW

**Comment by davidhunter08** [comment]
@edwardscull I've converted the GitHub issue templates to `.yaml` files.

File: docs/assets/css/app.scss [hunk 1]
```diff
@@ -45,3 +46,53 @@ pre {
     word-break: break-word;
   }
 }
+
+// Header overrides
+// Fixes service name placement and style
+.app-header {
```
Comment (edwardscull) [inline docs/assets/css/app.scss:52]: Should we split this code out into its own file?

Comment (davidhunter08) [reply docs/assets/css/app.scss]: Done @edwardscull

**Review by edwardscull — APPROVED** [review]
🎉

**Pull request merged**